docs/features/ValueTuples.cs

```diff
@@ -0,0 +1,537 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+
+internal static class Hash
```

Comment:
Probably not a great idea to put this at the top level - please place it inside a namespace.